docs/design/CLAUDE.md

```
@@ -0,0 +1,90 @@
+# Design Documents
+
+## Structure
+
+Each new design gets its own directory under `docs/design/`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)